po/de.po

```diff
@@ -166,7 +166,7 @@ msgstr "OK"
 #: src/app/qml/components/EditTagsDialog.qml:202
 #: src/app/qml/ui/EditNoteView.qml:604 src/app/qml/ui/NoteView.qml:150
 msgid "Close"
-msgstr "Schließen"
```

Comment:
Please do not change po files directly, but only on https://translate.ubports.com - otherwise we get merge conflicts that need to be resovled manually.

Reply:
I would do so, if notes-app was available on weblate. But I can not find it there. :-(
In alphabetical order it should be after music app, right?
![grafik](https://user-images.githubusercontent.com/37637312/47978846-3c036c00-e0c0-11e8-8e81-1df1cf91c5cd.png)